v1.1.1/js/Key.js

```diff
@@ -32,15 +32,11 @@
 		var code = KEY_CODES[event.keyCode];
 	    Key[code] = true;
 	    publish("key/"+code);
-	    event.stopPropagation();
-	    event.preventDefault();
 	}
 	Key.onKeyUp = function(event){
 		if(window.loopy && loopy.modal && loopy.modal.isShowing) return;
 		var code = KEY_CODES[event.keyCode];
 	    Key[code] = false;
-	    event.stopPropagation();
-	    event.preventDefault();
```

Comment:
So, it seems like removing either or both of the `event.stopPropagation()` or `event.preventDefault()` calls makes it so that if you do use a keyboard shortcut such as opening the inspector, it breaks the player. Specifically, once you close the inspector again and press play, you're then unable to click on the arrows displayed on the nodes to actually start the system